plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -798,6 +798,12 @@ private String buildImage(final DockerConnection dockerConnection,
                 Future<String> imageIdFuture = executor.submit(() -> {
                     ProgressStatus progressStatus;
                     while ((progressStatus = progressReader.next()) != null) {
+                        if (progressStatus.getError() != null) {
+                            String errorMessage = progressStatus.getError();
+                            if (errorMessage.matches("Error: image .+ not found")) {
```

Comment:
It will be another error that will be thrown in https://github.com/eclipse/che/pull/3084/files/8dd478fad8c533fd0a6a71e3bd37ebf2434ef3d3#diff-fca1a783572d79a80aa755d17f907b91R814